app/spaces/create/components/constant.tsx

```
@@ -0,0 +1,69 @@
+    const value = i.value
+    const label = i.label
+    switch(i.value) {
+        case 'general-group': {
+            icon = <UsersThreeIcon />
+            color = "#FFB672"
+            break;
+        }
+        case 'event-org': {
+            icon = <NewspaperClippingIcon />
+            color = "FFFA64"
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)